ecs-init/exec/iptables/iptables.go

```diff
@@ -61,38 +69,62 @@ func NewNetfilterRoute(cmdExec exec.Exec) (*NetfilterRoute, error) {
 
 // Create creates the credentials proxy endpoint route in the netfilter table
 func (route *NetfilterRoute) Create() error {
-	err := route.modifyNetfilterEntry(iptablesAppend, getPreroutingChainArgs)
+	err := route.modifyNetfilterEntry(iptablesTableNat, iptablesAppend, getPreroutingChainArgs)
+	if err != nil {
+		return err
+	}
+
+	err = route.modifyNetfilterEntry(iptablesTableFilter, iptablesInsert, getInputChainArgs)
 	if err != nil {
 		return err
 	}
-	return route.modifyNetfilterEntry(iptablesAppend, getOutputChainArgs)
+
+	return route.modifyNetfilterEntry(iptablesTableNat, iptablesAppend, getOutputChainArgs)
 }
 
 // Remove removes the route for the credentials endpoint from the netfilter
 // table
 func (route *NetfilterRoute) Remove() error {
-	preroutingErr := route.modifyNetfilterEntry(iptablesDelete, getPreroutingChainArgs)
+	preroutingErr := route.modifyNetfilterEntry(iptablesTableNat, iptablesDelete, getPreroutingChainArgs)
 	if preroutingErr != nil {
 		// Add more context for error in modifying the prerouting chain
-		preroutingErr = fmt.Errorf("Error removing prerouting chain entry: %v", preroutingErr)
+		preroutingErr = fmt.Errorf("error removing prerouting chain entry: %v", preroutingErr)
```

Comment:
nit `Error removing ...` <- this happens a couple more times below.  Keep the capitalization consistent between error messages.

Reply:
i think the error messages in this file are all lowercases i think? error level logs staring with cap is fine imo